src/radix_connect/well_known_client/models/dapp_definitions.rs

```diff
@@ -0,0 +1,67 @@
+use crate::prelude::*;
+
+/// Struct that represents content of well known file
+#[derive(Debug, Serialize, Deserialize, PartialEq, Clone, Eq)]
+#[serde(rename_all = "camelCase")]
+pub struct DappDefinitions {
```

Comment:
Maybe `DappWellKnownData/Info`? It should represent the well known that could contain more info in the future.